test/cases/vmware.sh

```diff
@@ -124,6 +124,10 @@ version = "0.0.1"
 [[packages]]
 name = "bash"
 
+[[packages]]
```

Comment:
Actually, I think it would be good to have a comment here with the rhbz# as well not just in the commit. Might be easier to find in the future.

Reply:
B/c I searched in `test/cases` and didn't find the file. Added.